### DISCUSSION TITLE
中文提问时出现prompt size exceeds the context window size问题

### DISCUSSION THREAD

**Discussion by Danielyy** [ymcui/Chinese-LLaMA-Alpaca#501] [Q&A]
PERSIST_DIRECTORY=db
MODEL_TYPE=GPT4All
MODEL_PATH=models/ggml-gpt4all-j-v1.3-groovy.bin
EMBEDDINGS_MODEL_NAME=sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2
MODEL_N_CTX=1000
TARGET_SOURCE_CHUNKS=4

在中文提问时出现错误
Enter a query: 李白的诗是什么风格？
ERROR: The prompt size exceeds the context window size and cannot be processed.GPT-J ERROR: The prompt is2940tokens and the context window is2048!

> Answer:
ERROR: The prompt size exceeds the context window size and cannot be processed.

**Comment by ymcui** [comment]
先参考：https://github.com/imartinez/privateGPT/issues/375
然后建议你去privateGPT那边找找解决方案。
看到你用的并不是本项目的模型，也不是太清楚相关出错原因。
